solution/backend/content_search/functions.py

```diff
@@ -130,3 +133,16 @@ def index_group(resources):
     '''
     for res in resources:
         add_to_index(res)
+
+
+def build_lambda_client():
+    if settings.USE_AWS_TOKEN:
+        # Assumes we are using credentials.  Works for local but an only post to an external URL.
+        return boto3_client('lambda',
```

Comment:
we need less credentials when running on the lambda to lambda than when we go local to aws.